DcShare/Controller/ShareViewController.swift

```diff
@@ -124,16 +130,21 @@ class ShareViewController: SLComposeServiceViewController {
     }
 
     override func configurationItems() -> [Any]! {
-        logger.debug("configurationItems")
-        // To add configuration options via table cells at the bottom of the sheet, return an array of SLComposeSheetConfigurationItem here.
-
         let item = SLComposeSheetConfigurationItem()
-        item?.title = String.localized("forward_to")
-        item?.value = selectedChat?.name
-        logger.debug("configurationItems chat name: \(String(describing: selectedChat?.name))")
-        item?.tapHandler = {
-            let chatListController = ChatListController(dcContext: self.dcContext, chatListDelegate: self)
-            self.pushConfigurationViewController(chatListController)
+        if isAccountConfigured {
+            logger.debug("configurationItems")
+            // To add configuration options via table cells at the bottom of the sheet, return an array of SLComposeSheetConfigurationItem here.
+
+
+            item?.title = String.localized("forward_to")
+            item?.value = selectedChat?.name
+            logger.debug("configurationItems chat name: \(String(describing: selectedChat?.name))")
+            item?.tapHandler = {
+                let chatListController = ChatListController(dcContext: self.dcContext, chatListDelegate: self)
+                self.pushConfigurationViewController(chatListController)
+            }
+        } else {
+            item?.value = String.localized("share_account_not_configured")
```

Comment:
if possible, i'd use `item?.title` here. the right-aligned gray text is easily overseen.

Reply:
done